neural_compressor/torch/quantization/algorithm_entry.py

```diff
@@ -67,6 +68,49 @@ def gptq_entry(
     return model
 
 
+###################### Static Quant Algo Entry ##################################
+@register_algo(STATIC_QUANT)
+@torch.no_grad()
+def static_quant_entry(
+    model: torch.nn.Module,
+    configs_mapping: Dict[Tuple[str, callable], StaticQuantConfig],
+    run_fn=None,
+    run_args=None,
+    example_inputs=None,
+    inplace=True,
+) -> torch.nn.Module:
+    logger.info("Quantize model with the static quant algorithm.")
+    # rebuild tune_cfg for static_quantize function
+    tune_cfg = {}
```

Comment:
```suggestion
    quant_config_mapping = {}
```
The algorithm implementation is not aware of the tuning concept.